findbugs-filter.xml

```diff
@@ -3,5 +3,8 @@
     <Match>
         <Class name="~com\.microservicesteam\.adele\..+\.domain\.Immutable.*" />
     </Match>
+    <Match>
+        <Class name="~com\.microservicesteam\.adele\..+\.model\.Immutable.*" />
```

Comment:
I only spotted this as inconsistency and I was curious about the reasons behind this. Let's discuss this in person.